src/network.c

```diff
@@ -75,6 +75,8 @@ static gboolean transfer(RaucTransfer *xfer, GError **error)
 	char errbuf[CURL_ERROR_SIZE];
 	gboolean res = FALSE;
 
+	g_return_val_if_fail(error == NULL || *error == NULL, FALSE);
```

Comment:
If we add an input check for `error`, we should add one for `xfer`, too, to remain consistent.

``` diff
+	g_return_val_if_fail(xfer, FALSE);
```

Reply:
Done.